src/test/java/io/vertx/ext/mongo/MongoClientTest.java

```diff
@@ -117,13 +139,15 @@ private void testFindBatch(BiFunction<CountDownLatch, ReadStream<JsonObject>, Li
     awaitLatch(latch);
     assertEquals(numDocs, foos.get().size());
     assertEquals("bar0", foos.get().get(0));
-    assertEquals("bar999", foos.get().get(numDocs - 1));
-
+    String bar = "bar";
+    for(int i=0;i<String.valueOf(numDocs).length()-1;i++){
```

Comment:
`String.valueOf(numDocs).length()` Isn't that just `4`?

Reply:
in one case it's 4 in one case its 3.